numpy/core/shape_base.py

```diff
@@ -206,6 +207,7 @@ def vstack(tup):
     dstack : Stack arrays in sequence depth wise (along third dimension).
     concatenate : Join a sequence of arrays along an existing axis.
     vsplit : Split array into a list of multiple sub-arrays vertically.
+    block : Create block arrays.
```

Comment:
"Create block arrays" is a bit ambiguous, I would recommend a couple of alternatives:

  - Join arrays along multiple axes
  - Assemble arrays from blocks

Same goes for line 226